examples/research_projects/onnx/summarization/bart_onnx/generation_onnx.py

```diff
@@ -625,7 +625,7 @@ def _reorder_cache(self, past: List[torch.Tensor], beam_idx):
         # speedy decoding is disabled and no need to reorder
         reordered_decoder_past = []
         for state in past:
-            reordered_decoder_past.append(state.index_select(0, beam_idx))
+            reordered_decoder_past.append(state.index_select(0, beam_idx.to(state.device)))
```

Comment:
Let's not include examples and especially onnx related stuff, not sure if it is needed but it's certainly not tested 😉 
```suggestion
            reordered_decoder_past.append(state.index_select(0, beam_idx))
```